PublicComponentCode/ComponentCode/src/java/main/ccsu/edu/grovepicomponents/GrovePiFan.java

```diff
@@ -0,0 +1,120 @@
+package ccsu.edu.grovepicomponents;
```

Comment:
Package naming should be in format reverse of domain names, least specific getting more specific.  So edu.ccsu.grovepicomponents.  Note this is particularly a bit odd since you have packages with both "ccsu.edu" and "edu.ccsu"